servicebroker_node/lib/api/api.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function() {
+
+  var fs = require('fs');
+  var path = require('path');
+
+  var settings = JSON.parse(
```

Comment:
ok, we did hardcoded the service details and plan details with assumption that we only have 1 plan and 1 service id here.  Maybe back to prior JSON stream approach (save the catalog information in a JSON, and then read it directly) is better.